src/main/resources/assets/carpet/scripts/draw.sc

```diff
@@ -0,0 +1,196 @@
+import('math','_euclidean_sq','_vec_length');
+
+__config() -> {
+    'commands'->{
+        'sphere <center> <radius> <block>'->['draw_sphere',null, false],
+        'sphere <center> <radius> <block> replace <replacement>'->['draw_sphere',false],
+        'ball <center> <radius> <block>'->['draw_sphere', null, true],
+        'ball <center> <radius> <block> replace <replacement>'->['draw_sphere', true],
+        'diamond <center> <radius> <block>'->['draw_diamond',null],
+        'diamond <center> <radius> <block> replace <replacement>'->'draw_diamond',
+        'cone <center> <radius> <height> <pointing> <orientation> <block> <hollow>'->['draw_pyramid',null, false],
```

Comment:
you can even replace initial selection of shapes into a term argument <shape> - that would reduce it to two lines

Reply:
what is that?

Reply:
ah ok I see.

Reply:
wait how? they have differing arg numbers.

Reply:
ye I tried, and id dont work.

Reply:
@gnembon any suggestions?

Reply:
yeah, different arg numbers - need to stay this way.